client/integrationTest/integration.test.ts

```diff
@@ -558,6 +573,10 @@ describe('Integration Test', () => {
                 javascriptTestRunResult = await runJavaScriptSmartContractTests(smartContractName);
             } else if (language === 'TypeScript') {
                 await generateSmartContractTests(smartContractName, '0.0.1');
+
+                const errorSpy: sinon.SinonSpy = mySandBox.spy(vscode.window, 'showErrorMessage');
+                await submitTransaction(smartContractName, '0.0.1', 'transaction1', 'hello world');
+                errorSpy.should.not.have.been.called;
```

Comment:
Can we put this further down? if we're not expecting error spy to be called at all during this test?